app/models/qa/authorities/local/find_works.rb

```diff
@@ -0,0 +1,23 @@
+module Qa::Authorities
```

Comment:
Or  `app/authorities/sufia/authorities/local/find_works.rb` if you care going to put it in the `Sufia::Authoritities::Local` namespace.